### PR TITLE
Make card images clickable

### PR DESCRIPTION
**Pull request by tversteeg**
Card images will be a hyperlink to the primary URL if available.

### CODE REVIEW

**Comment by ozkriff** [comment]
Checked locally and something seems to be work not as intended:

image

**Comment by 17cupsofcoffee** [comment]
Ah, this is probably to do with how the CSS for the card images is implemented. I'll have a look at how it can be fixed.

File: templates/categories/macros.html [hunk 1]
```diff
@@ -45,9 +45,15 @@
...
     {% if item.image %}
+        {% if primary_url %}
+            <a href="{{ primary_url }}">
+        {% endif %}
         <div class="image">
             <img src="{{ item.image }}" alt="Picture of {{ name }}">
         </div>
+        {% if primary_url %}
+            </a>
+        {% endif %}
```
Comment (17cupsofcoffee) [inline templates/categories/macros.html:56]: Due to the way Semantic UI works (i.e: badly), the anchor tag needs to replace the `div` rather that surround it.

```suggestion
         {% if primary_url %}
            <a class="image" href="{{ primary_url }}">
        {% else %}
            <div class="image">
        {% endif %}
            <img src="{{ item.image }}" alt="Picture of {{ name }}">
        {% if primary_url %}
            </a>
        {% else %}
            </div>
        {% endif %}
```

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**